concepts/modularization/enable_disable_module.md

```diff
@@ -25,4 +31,21 @@ Similarly, to disable a module, run:
    kyma alpha disable module {MODULE_NAME}
    ``` 
 
+</details>
+<details>
+<summary label= Kyma Dashboard>
+Kyma Dashboard
+</summary>
+
+1. Go to the `kyma-system` Namespace.
+2. Choose the **Kyma** resource from the **Kyma** section.
```

Comment:
I'd revert the order so that *where* it happens comes first, and then what needs to be done.

```suggestion
2. In the **Kyma** section, choose the **Kyma** resource.
```